client/layout/guided-tours/tours/simple-payments-email-tour.js

```diff
@@ -19,6 +19,26 @@ import {
 	Quit,
 } from 'layout/guided-tours/config-elements';
 import { AddContentButton } from '../button-labels';
+import { getSectionName } from 'state/ui/selectors';
+
+const sectionHasSidebar = state =>
```

Comment:
Is the selector `hasSidebar` in `client/state/ui/selectors.js` incorrect?

Reply:
Apparently it is incorrect at least for Customizer (where technically the sidebar exists, but we don't want this tour to be visible there).

Reply:
Huh okay, let's keep this here for now, but we may consider adding a global selector that works in a follow up PR.

Reply:
The fact is, my whitelist is not perfect (e.g. it won't be updated when new sections are added), but it's good enough for our purposes here (avoid displaying the tour when there is no sidebar, in as many cases as possible).

Ideally we should improve the `hasSidebar` selector to be bulletproof.
That'd probably require an audit of all the sections, and figure out why some of them (e.g. Customizer) are set with the sidebar.

Reply:
> Ideally we should improve the hasSidebar selector to be bulletproof.

Until then would it be better to rely on `hasSidebar` and account for exceptions? It seems like that would be less likely to break when new sections are added. Eg. `hasSidebar && ! includes( [ 'customizer', ... ], getSectionName( state ) )`

Reply:
Makes sense, I'll update asap.